core/driver/sources/src/main/scala/org/virtuslab/ideprobe/reporting/ErrorValidator.scala

```diff
@@ -18,8 +20,13 @@ object ErrorValidator {
     }
   }
 
+  private def ideMessageMatchesMessagesFromConfig(ideMessage: IdeMessage, messagesFromConfig: Seq[String]): Boolean =
+    messagesFromConfig.exists(configMessage => trimEachLine(ideMessage.content).contains(trimEachLine(configMessage)))
+
+  private def trimEachLine(s: String): String = s.split('\n').map(_.trim).mkString("\n")
+
   private def toString(errors: Seq[IdeMessage]): String = {
-    val sb = new StringBuilder()
```

Comment:
Hmmm was it a different `StringBuilder` class beforehand? before the explicit import has been added

Reply:
No, it has been the same. Added explicit import as IDE suggested. IMO it's nice to get rid of IDE alerts if possible